ports/stm/boards/nucleo_f746zg/mpconfigboard.h

```diff
@@ -34,6 +34,7 @@
 #define FLASH_PAGE_SIZE             (0x4000)
 
 #define BOARD_OSC_DIV (8)
+#define BOARD_HAS_LOW_SPEED_CRYSTAL (0)
```

Comment:
The nucleo f746zg has a LSE crystal installed, but not a HSE.

Reply:
I think it just grabs the 8MHz crystal from the STlink, like most ST dev boards, which seems to be identical as far as clock settings go.

Reply:
That's true...but it does have a crystal installed. X2
source: https://www.st.com/en/evaluation-tools/nucleo-f746zg.html (and my eyes..as I look at the board)

Reply:
Yep! Didn't mean to contest your correction about the 32k crystal, I mixed it up with another board. Working on testing locally.